wysadzulice/assets/js/new_campaign.js

```diff
@@ -0,0 +1,40 @@
+import jquery from 'jquery';
+import Planting from 'plantingjs';
+import { View } from 'backbone';
+
+
+export default View.extend({
+
+  initialize: function(options) {
+    this.$el = jquery('.viewport');
+    this.manifestoUrl = options.manifestoUrl;
+    this.googleApiKey = options.googleApiKey;
+    this.saveUrl = options.saveUrl;
+    this.render();
+  },
+
+  render: function() {
+    return new Planting( {
+      container: this.$el,
+      manifestoUrl: this.manifestoUrl,
+      googleApiKey: this.googleApiKey,
+      saveUrl: this.saveUrl,
+      selectPanoMode: true,
+      onSelectPano: this.on_select_piano.bind(this),
+    });
+  },
+
+  on_select_piano: function(campaign) {
```

Comment:
We need to separate this code from the view. Basically this is what `Backbone.Model` does. For example you can modify it this way

```javascript
{
  initialize: (options) {
    this.model = new Backbone.Model({ url: options.saveUrl });
    this.model.on('change', this.model.save);
    /* ... */
    new Planting({
      onSelectPano: this.model.set
    })
  }
}
```

If you need more customization of saving process you can override `Backbone.Model.prototype.save` or `Backbone.Model.prototype.sync`.